module/User/src/Model/User.php

```diff
@@ -152,13 +152,34 @@ public function getRoleNames()
     }
 
     /**
-     * Get the user's role ID.
-     *
      * @return string
      */
-    public function getRoleId()
+    public function getRoleId(): string
     {
-        return 'user_' . $this->getLidnr();
+        $roleNames = $this->getRoleNames();
+        if (in_array('admin', $roleNames)) {
+            return 'admin';
+        }
+
+        if (in_array('company_admin', $roleNames)) {
+            return 'company_admin';
+        }
```

Comment:
If a board member has the `company_admin` role (i.e. they are part of C4) they do not get the correct board privileges.

Reply:
They simply shouldn't be I'll come up with a way that applies this assertion to all roles instead.